assets/scss/_colors.scss

```diff
@@ -0,0 +1,6 @@
+// Colors
+$bg-color: #fefefe !default;
+$fg-color: #323232 !default;
+$alt-bg-color: #dcdcdc !default;
+$alt-fg-color: #000 !default;
+$link-color: #3366CC !default;
```

Comment:
This file will conflict with my styling refactor. But, don't mind, I will fix it.

Reply:
It is just a matter of renaming this file. Don't worry.
I created a similar file in my `style-refactor` branch. 

https://github.com/luizdepra/hugo-coder/blob/style-refactor/assets/scss/_variables.scss

Reply:
That's really cool. I actually wanted to suggest similar thing, and also add an empty `_override.scss` file in the theme and include it in `coder.scss` which can be used by the user to _override_ from their site.